### ISSUE TITLE
New rule: Scrollable element is keyboard accessible

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#484]
The ACT-Rules community group has completed work on {{rule name}}. We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: Scrollable element is keyboard accessible
**Rule id**: 0ssw9k
**Link to the rule**: https://act-rules.github.io/rules/0ssw9k
**License**: W3C Software And Document Notice And License
**Implementors**:
- QualWeb
- SortSite

If you'd like to contact me, please e-mail me at wilcofiers@deque.com. You can also reach me on the W3C Community Slack, my name there is Wilco Fiers

**Comment by maryjom** [comment]
[Survey](https://www.w3.org/2002/09/wbs/93339/ACTScrollableElement/) is open until 8 October.

**Comment by maryjom** [comment]
Looks like some things needed updating. Discussed survey results during the following meetings:

- [24 September minutes](https://www.w3.org/2020/09/24-wcag-act-minutes.html#item02)
- [8 October minutes](https://www.w3.org/2020/10/08-wcag-act-minutes.html#item02)

**Comment by WilcoFiers** [comment]
Updates were done a while ago. Apologies for missing this. Since this predates our policy of editorial changes going directly to CFC this is going to have to go on the agenda again.

**Comment by WilcoFiers** [comment]
I mislabeled this one. I think we need to touch on this update on the next call, and decide if we want it on another survey, or if it can go to CFC.

**Comment by WilcoFiers** [comment]
Rule is in CFC until Feb 16th

**Comment by woody-li** [comment]
I'm not sure whether appropriate to ask questions here.

For display dynamic content in a overflow: auto element, how to controls the accessible (maybe tabindex)?

Examples:
- Some default closed `details` elements , box element don't show the scrollbar, the scrollbar displays when they are opened.
- Box element don't show the scrollbar when browser maximize, how about responsive?

Should I must add a resize listener to update the accessible state?
